SampleCode.cs

```diff
@@ -127,14 +127,15 @@ private static void ShowMethods()
             Console.WriteLine("    GetAnAcceptPaymentPage");
             Console.WriteLine("    GetCustomerProfileTransactionList");
             Console.WriteLine("    GetAccountUpdaterJobSummary");
+            Console.WriteLine("    GetAccountUpdaterJobDetails");
         }
 
         private static void RunMethod(String methodName)
         {
             // These are default transaction keys.
             // You can create your own keys in seconds by signing up for a sandbox account here: https://developer.authorize.net/sandbox/
-            const string apiLoginId = "5KP3u95bQpv";
-            const string transactionKey = "346HZ32z3fP4hTG2";
+            const string apiLoginId = "mbld_api_-Bgb1Oig";
```

Comment:
Please remove these credentials. The original credentials work in the Sandbox environment. The new credentials are specific to color.

Reply:
Removed the credentials and kept  original credentials.